CharlieBackend.AdminPanel/Controllers/ThemesController.cs

```diff
@@ -56,14 +47,15 @@ public async Task<IActionResult> AddTheme(CreateThemeDto themeDto)
             return RedirectToAction("AllThemes", "Themes");
         }
 
-        [HttpPost("{id}")]  
-        public async Task<IActionResult> UpdateTheme(long id, UpdateThemeDto data)
+        [HttpGet("{id}")]  
```

Comment:
Why are you using GET method on update logic?
Probably should be better to use HttpPut?

Reply:
unfortunately it`s not possible, because we can`t use HttpPut  in this case

Reply:
<a> tag supports only get method